.circleci/config.base.yml

```diff
@@ -149,6 +149,7 @@ jobs:
           steps:
             - install_node_windows
             - checkout
+            - run: sed -i '/24e2b385b6510538b4253aba62f243f7a1783fa544970c0712bcc3dd3c10159f7eab741f8dfaf409afed5ed8e0c334ea2c046ffbd11b93901c41847142a34b5e/d' yarn.lock
```

Comment:
What is this for?

Reply:
yarn adds the checksum for pkg-fetch for validation, pkg-fetch has different checksum for windows/linux package causing the validation to fail. This line removes the checksum from yarn. Lock and prevents windows test from failing on checksum validation

Reply:
yeah. that's going to be a hidden trap. perhaps this could be a `/script/remove_pkg_fetch_checksum.ts` that doesn't assume hash value but just looks for this in the file.